Bookshelf/settings.gradle.kts

```diff
@@ -1,3 +1,5 @@
+// If you want to run against the local source repository just include the source projects using
+// includeBuild("../../packages")
```

Comment:
Did this just work for you? I had to do numerous updates to the SDK project to get it to work (fixing JDK version, updating AGP, commenting out some JSON stuff in the analytics part). Don't think we should included it unless it works out of the box.

Reply:
I didn't test it, so probably a good idea to remove it